generators/firechip/src/main/scala/TargetConfigs.scala

```diff
@@ -99,6 +82,23 @@ class WithFireSimConfigTweaks extends Config(
   new chipyard.config.WithNoDebug
 )
 
+// Tweaks that are generally applied to all firesim configs
+class WithFireSimConfigTweaks extends Config(
+  // Optional*: Removing this will require adjusting the UART baud rate and
```

Comment:
Like this change.
Maybe call this `WithFireSimClockingTweaks`, and keep the name of the old one?

Reply:
Actually, its probably safer to rename the old one as well, `WithFireSimDesignTweak`, perhaps?

Reply:
This `WithFireSimConfigTweaks` combines the clocking with the "design" tweaks. In order to avoid changing FireSim itself (changing the build recipes), I will at least keep this name `WithFireSimConfigTweaks`.

Reply:
Could  you make a `WithFireSimDefaultFrequencies` or something and compose the two of them to produce the old tweaks.

Reply:
I had to unresolve this.